packages/wallets/src/evm/connectors/smart-wallet/index.ts

```diff
@@ -216,6 +237,22 @@ export class SmartWalletConnector extends Connector<SmartWalletConnectionArgs> {
     return await this.accountApi.isAcountDeployed();
   }
 
+  /**
+   * Get the underlying account smart contract of the smart wallet.
+   * @returns the account smart contract of the smart wallet.
+   */
+  async getAccountContract(): Promise<Account<IAccountCore>> {
+    const signer = await this.getSigner();
+    const network = (await (await this.getProvider()).getNetwork()).chainId;
+    const sdk = ThirdwebSDK.fromSigner(signer, network);
```

Comment:
Was there no way to get the contract from account API ? Might need to update the signer on it but otherwise it should work.

That way it inherits the options passed in like abi